techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -55,6 +55,14 @@ bundle agent computeInventoryTime
 
 bundle agent doInventory
 {
+  classes:
+    "disable_inventory" expression => fileexists("${g.rudder_disable_inventory_file}");
+  methods:
+    !disable_inventory::
+      "any" usebundle => doInventory_always;
+}
+bundle agent doInventory_always
```

Comment:
The force_inventory is used twice, once in the"rudder agent inventory" command and in the packaging to run a normal inventory after an install or an upgrade.

In those case, the inventory should not be run if the disable-inventory file is present.

If we want to make this coherent with disable-agent, we should have a new parameter "rudder agent inventory --force" and split the force_inventory class into run_inventory_now and (really_)force_inventory